src/index.html

```diff
@@ -11,7 +11,143 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    method="post"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+  >
```

Comment:
```suggestion
    <form
      method="post"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
    >
```
The closing tag should be under the opening. Also, pay attention to indentations